base/dcp_feed_type.go

```diff
@@ -51,6 +143,12 @@ type SGFeedSourceParams struct {
 
 	// Used to pass the SG database name to SGFeed* shims
 	DbName string `json:"sg_dbname,omitempty"`
+
+	// Scope within the bucket to stream data from.
+	Scope string `json:"scope,omitempty"`
```

Comment:
Just for my own benefit - does GoCB/CBGT not allow streaming >1 scope?

Reply:
I believe so:

https://github.com/couchbase/cbgt/blob/91cc2647a6b9cb775e476319ab52b5baf79d85a0/feed_dcp.go#L84-L88